### PR TITLE
(ja) additional Schema translations

### PR DESCRIPTION
**Pull request by fisuda**
Hi @jason-fox,
I made additional Schema translations. It will be great if you could review this PR.

Thanks.

### CODE REVIEW

File: schema/ja/Product/index.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    "size":  {
+      "@id": "tutorial:size",
+      "@type": "xsd:string"
+    }
+  },
+  "@graph": [
+    {
+      "@id": "tutorial:Product",
+      "@type": "rdfs:Class",
+      "rdfs:comment": "Product is something sold in a Store.",
```
Comment (jason-fox) [inline schema/ja/Product/index.html]: rdfs:comment and label should be Japanese

File: schema/ja/Shelf/index.html [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+    "stocks": {
+      "@id": "tutorial:stocks",   
+      "@type": "@id"
+    }
+  },
+  "@graph": [
+    {
+      "@id": "tutorial:Shelf",
+      "@type": "rdfs:Class",
+      "rdfs:comment": "A Shelf is a unit of Store Furniture",
```
Comment (jason-fox) [inline schema/ja/Shelf/index.html]: rdfs:comment and label should be Japanese throughout

File: schema/ja/StockOrder/index.html [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+    "stockCount":  {
+      "@id": "tutorial:stockCount",
+      "@type": "xsd:integer"
+    }
+  },
+  "@graph": [
+    {
+      "@id": "tutorial:StockOrder",
+      "@type": "rdfs:Class",
+      "rdfs:comment": "StockOrder an entry in a list of items sold in one or more stores",
```
Comment (jason-fox) [inline schema/ja/StockOrder/index.html]: rdfs:comment and label should be Japanese throughout

File: schema/ja/StockOrder/index.html [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+        {
+          "@id": "tutorial:StockOrder"
+        }
+      ],
+      "schema:rangeIncludes": [
+        {
+          "@id": "fiware:Building"
+        }
+      ],
+      "rdfs:comment": "Store for which an item is requested",
```
Comment (jason-fox) [inline schema/ja/StockOrder/index.html]: rdfs:comment and label should be Japanese

File: schema/ja/Store/index.html [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+        {
+          "@id": "fiware:Building"
+        }
+      ],
+      "schema:rangeIncludes": [
+        {
+          "@id": "tutorial:Shelf"
+        }
+      ],
+      "rdfs:comment": "Units found within a Building",
```
Comment (jason-fox) [inline schema/ja/Store/index.html]: rdfs:comment and label should be Japanese

**Review by jason-fox — CHANGES_REQUESTED** [review]
That's great, but I would like one further consistent change.

Within the `@graph` of the snippets for Store. Shelf, StockOrder etc, the  `rdfs:comment` and `label` should be translated to Japanese as well.

The root file `tutorials-context.jsonld` is *properly* bilingual, but the snippets are just for teaching and extra English would get in the way of the teaching goal.

In a real-world example the human-readable schema would be generated from the `@graph` anyway.

As an aside - i see that most of my attempts at auto-translated Japanese were pretty terrible. Thanks for correcting them.

**Comment by fisuda** [comment]
I have updated them.

File: schema/ja/Shelf/index.html [hunk 6]
```diff
@@ -180,8 +180,8 @@
           "@id": "schema:Person"
         }
       ],
-      "rdfs:comment": "Object requested by person.",
-      "rdfs:label": "installedBy"
+      "rdfs:comment": "Person who installs an Object",
```
Comment (jason-fox) [inline schema/ja/Shelf/index.html]: One missing here

**Person who installs an Object** should be **オブジェクトを設置する人** . I guess.

**Review by jason-fox — APPROVED** [review]
Could you make the final change and I'll merge.

**Comment by fisuda** [comment]
I have fixed it.

**Comment by fisuda** [comment]
Thank you for your support despite the weekend.

**Pull request merged**